src/components/webview/WebviewResolver.ts

```diff
@@ -12,6 +12,7 @@ export class WebviewResolver {
 				Uri.joinPath(this.__extensionPath, 'intuita-webview/build'),
 				Uri.joinPath(this.__extensionPath, 'resources'),
 			],
+			retainContextWhenHidden: true,
```

Comment:
what does that do?

Reply:
this keeps webview active when we switch to other tabs, if set to false webview will unmount each time